Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -52,6 +53,13 @@ class HistoryPanel: UIViewController, HomePanel {
         return button
     }()
 
+    func fetchSyncedDevicesCount() -> Success {
+        return chainDeferred(self.profile.getCachedClientsAndTabs()) { deferMaybe($0.count) }.bindQueue(dispatch_get_main_queue()) { result in
+            self.currentSyncedDevicesCount = result.successValue
```

Comment:
Nit: If you're just updating the value and not touching the UI you don't need to be on the main thread. You can update the count in your `chainDeferred` block.

```swift
return chainDeferred(self.profile.getCachedClientsAndTabs()) { tabsAndClients in
    self.currentSyncedDevicesCount = tabsAndClients.count
    return succeed()
}
```